code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -49,6 +54,20 @@
 
 	var/obj/item/radio/integrated/signal/sradio // AI's signaller
 
+	var/canholo = 1
+	var/obj/item/weapon/card/id/access_card = null
+	var/chassis = "repairbot"
+	var/list/possible_chassis = list("cat", "mouse", "monkey", "corgi", "fox", "repairbot")

```

Comment:
Do not initialize lists in here, do it in `/new` or in another proc that requires this list by checking if it's `null` and then setting the list up.

See: #21876